samples/matter/closure/boards/nrf54l15dk_nrf54l15_cpuapp_ns.overlay

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	aliases {
```

Comment:
This file can be removed, we don't need to support NS configuration with nRF54L15